pinot-common/src/main/proto/plan.proto

```diff
@@ -43,7 +43,9 @@ message StageNode {
   int32 stageId = 1;
   string nodeName = 2;
   repeated StageNode inputs = 3;
-  ObjectField objectField = 4;
+  repeated string columnNames = 4;
+  repeated string columnDataTypes = 5;
```

Comment:
i incline to make columnDataTypes ordinal enabled enum (to store as integer) instead of storing string here. but not sure if this could cause some backward-incompatible changes to DataSchema.ColumnDataType. 

Please share your feedback